packages/react/src/components/editor/JSONEditor.tsx

```diff
@@ -53,7 +53,10 @@ export interface JSONEditorProps {
 }
 
 export interface JSONEditorStateProps {
-    value: string;
+    /**
+     * @deprecated use defaultValue instead
+     */
+    value?: string;

```

Comment:
The value is already optional and deprecated in `JSONEditorProps`. If you apply my change on the JSONEditorConnected function I don't think you need this change

```suggestion
    value: string;
```